Migrated documentation for text inputs (#367)
indiana-university/rivet-source: src/components/text-input/README.md

# Text input

Text inputs are the basic building blocks of forms. They allow users to enter various types of data into web-based forms.

## When to use

> Use text inputs for information that is limited to a single line. This can include:

Reviewer: ```suggestion
Use text inputs for input fields that are limited to a single line. Some examples include:
```

- Name
- Phone number
- City

## When to use something else

> If the information you require includes multiple sentences or lines, a textarea is more appropriate.

Reviewer: ```suggestion
If the input field will require multiple sentences or lines, a textarea is more appropriate.
```
